### PR TITLE
KUR-55 Thunder Liveblogging: Final adjustments

### PR DESCRIPTION
**Pull request by milkovsky**
UI changes:
 Move "Liveblog notification settings" to Content Authoring and rename to "Liveblog" - so any further general configuration can be added there in future
 Move "Liveblog post lists" to Content -> Tab: "Liveblog posts" (similar to comment, files, media ...)
 Improve wording in texts: No entity, consistent casing with core.

 Check how to handle fields
 Add LiveblogPostInterface
 Use display configuration to hide uid and liveblog from form
 Improve 'Status' " Whether post is published. "  checkbox wording based upon what core has in d7
 Remove useless description "The title of the liveblog post. "
 Display config of paragraphs field is not working.
 "Trigger an notification channel message." should be done via entity hooks, not via the form
 Messages: Make sure they are shown with and without ajax.
 Make sure the field is indexed (REST page, the query with liveblog posts list).

Update all the configurations(remove unused views, etc.)
Test fresh install

### CODE REVIEW

**Comment by goldquest** [comment]
Is there an issue for KUR-55?

File: liveblog.module [hunk 1]
```diff
@@ -182,3 +182,33 @@ function liveblog_js_settings_build(&$settings, \Drupal\Core\Asset\AttachedAsset
   }
   $settings['liveblog']['libraries'] = $libraries;
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_insert().
+ */
+function liveblog_liveblog_post_insert(\Drupal\liveblog\Entity\LiveblogPost $entity) {
+  liveblog_trigger_notification_channel_liveblog_post_event($entity, 'add');
```
Comment (fago) [inline liveblog.module]: Should be LiveblogPostEntity::postSave()

File: src/Entity/LiveblogPost.php [hunk 2]
```diff
@@ -353,13 +309,7 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setSetting('target_type', 'user')
       ->setDefaultValueCallback('Drupal\node\Entity\Node::getCurrentUserId')
       ->setDisplayOptions('form', [
```
Comment (fago) [inline src/Entity/LiveblogPost.php]: It would be nice to allow configuration of the author form widget if people want to.

Comment (milkovsky) [inline src/Entity/LiveblogPost.php]: they can indeed do so. See below: `->setDisplayConfigurable('form', TRUE)`

hidden is a default

**Comment by milkovsky** [comment]
Let's merge?

**Pull request merged**